src/I18n/Formatter/IcuFormatter.php

```diff
@@ -35,7 +35,7 @@ class IcuFormatter implements FormatterInterface
      * @throws \Aura\Intl\Exception\CannotFormat
      * @throws \Aura\Intl\Exception\CannotInstantiateFormatter
      */
-    public function format($locale, $message, array $vars): string
+    public function format($locale, $message, $vars): string
```

Comment:
whats wrong with this one?

Reply:
Nothing, removed mistakenly. The perils of PHP 7.2 parameter type widening.